administration/package.json

```diff
@@ -35,36 +35,47 @@
     "@graphql-codegen/typescript": "2.7.3",
     "@graphql-codegen/typescript-operations": "2.5.3",
     "@graphql-codegen/typescript-react-apollo": "3.3.3",
-    "@testing-library/jest-dom": "^5.16.5",
     "@types/apollo-upload-client": "^17.0.2",
     "@types/blob-stream": "^0.1.30",
-    "@types/jest": "^29.2.4",
+    "@types/jasmine": "^4.3.1",
     "@types/node": "^18.7.18",
     "@types/react": "^18.0.20",
     "@types/react-dom": "^18.0.6",
     "@types/react-router-dom": "5.3.3",
     "@types/styled-components": "^5.1.26",
     "apollo": "^2.34.0",
     "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-jasmine": "^4.1.3",
+    "jasmine": "^4.5.0",
+    "karma": "^6.4.1",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-jasmine": "^5.1.0",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma-webpack": "^5.0.0",
     "prettier": "^2.7.1",
+    "ts-loader": "^9.4.2",
     "typescript": "^4.8.3"
   },
   "scripts": {
     "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts start",
     "build": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts build",
-    "test": "react-scripts test",
+    "test": "karma start --single-run",
```

Comment:
We should eject sooner or later. I think we are still depending on jest through react-scripts.

Reply:
Yeah, I agree... We are slowly but surely moving away from react-scripts :)